src/api/spec/features/webui/main_page_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "browser_helper"
+
+RSpec.feature "OBS main page", :type => :feature, :js => true do
+  let(:admin_user) { create(:admin_user) }
+  let(:user) { create(:confirmed_user) }
+
+  describe "#index" do
```

Comment:
Okay. I'll remove the context block. And "OBS main page" is already there because of the RSpec.feature definition